pkg/ccl/backupccl/restore_span_covering_test.go

```diff
@@ -270,7 +271,7 @@ func makeImportSpans(
 
 	filter, err := makeSpanCoveringFilter(
 		checkpointFrontier,
-		nil,
+		highWaterMark,
 		introducedSpanFrontier,
 		targetSize,
 		true)
```

Comment:
the highWaterMark will not get exercised unless that last arg that is currently set to `true` is changed. We should set it to false if the HWM is not empty.